styles/widgets/generic/radioButton.generic.less

```diff
@@ -89,6 +89,15 @@
     }
 }
 
+
+.dx-radiogroup:not(.dx-state-disabled) {
```

Comment:
Can you get rid of `not` selector and use styles for `.dx-radiobutton .dx-state-disabled` and `.dx-radiogroup.dx-state-disabled`? We want to make selectors weakly not only in dxTreeView but in other widgets too.